code/environment.yml

```diff
@@ -0,0 +1,21 @@
+name: AstronomicalData
+channels:
+  - default
+  - conda-forge
+  - astropy
+dependencies:
+  - python>=3.6
+  - jupyter
+  - numpy
+  - matplotlib
+  - seaborn
+  - pandas
+  - pytables
+  - h5py
+  - scipy
+  - astropy
+  - astroquery
+  - gsl
```

Comment:
I'm curious about how we are using gsl? Is this a dependency from one of the other tools?

Reply:
I don't think we're using it. I'm not sure where that line came from, but I don't think it was me.